sections/popular-products.liquid

```diff
@@ -0,0 +1,139 @@
+{{ 'popular-products.css' | asset_url | stylesheet_tag }}
+<h3 class="title title--popular-products">Lo más popular</h3>
+<div class="popular-products__container">
+  <div class="product-block">
+    {% for block in section.blocks %}
+      <div class="product-block__container">
+        <div class="product-block__img-container">
+          <img
+            src="{{ block.settings.product.images[0] | img_url: 'master' }}"
```

Comment:
As I wrote in #2 and #3, `img_url` is deprecated and **must not be used**

https://shopify.dev/docs/api/liquid/filters/img_url

-----
<img width="796" alt="image" src="https://github.com/Luis-Esquivia/analyticstheme-shopify/assets/16655413/9d93e2e6-9afd-4f72-a7ae-f3e4f2006d9e">

```suggestion
            src="{{ block.settings.product.images[0] | image_url: width: 210, height: 170 }}"
```